src/H5.c

```diff
@@ -101,32 +100,24 @@ static H5_atclose_node_t *H5_atclose_head = NULL;
 /* Declare a free list to manage the H5_atclose_node_t struct */
 H5FL_DEFINE_STATIC(H5_atclose_node_t);
 
-/*--------------------------------------------------------------------------
-NAME
-    H5__init_package -- Initialize interface-specific information
-USAGE
-    herr_t H5__init_package()
-RETURNS
-    Non-negative on success/Negative on failure
-DESCRIPTION
-    Initializes any interface-specific data or routines.
---------------------------------------------------------------------------*/
```

Comment:
There's a lot to dislike about our function comments, but please don't change the coding style without a plan to propagate the new paradigm throughout the library.

Reply:
I have added a function comment I found elsewhere to _init routines that were missing them.